product-service/addProduct.js

```diff
@@ -0,0 +1,65 @@
+const { Client } = require('pg');
```

Comment:
better to use pool.
take a look here https://teams.microsoft.com/l/message/19:73dafd02206d421896229364386d7bd2@thread.skype/1627078337861?tenantId=b41b72d0-4e9f-4c26-8a69-f949f367c91d&groupId=a91a4b92-98b1-4d0d-9e21-bd706bd68ed5&parentMessageId=1627078337861&teamName=EPAM%20JS%20EDU%20L%26D&channelName=NodeJS%20in%20AWS%20Cloud&createdTime=1627078337861